website/content/docs/backends/slurm.md

```diff
@@ -0,0 +1,48 @@
+---
+title: SLURM
+
+menu:
+  main:
+    parent: backends
+    weight: 20
+---
+
+# SLURM
+
+Funnel can be configured to start workers via [SLURM][slurm].  
+
+Workers will start, execute a single task, then exit immediately so that they don't
+unfairly hold slots in the SLURM queue. Funnel accesses SLURM by making calls
+to `sbatch`.
+
+The Funnel server process needs to run on the same machine as the SLURM master.  
+Configure Funnel to use SLURM by including the following config:
+
+```YAML
+Scheduler: slurm
+Backends:
+    SLURM: |
+        #!/bin/bash
```

Comment:
If it's easy to import these from a file in Hugo, it will prevent them from getting out of sync. It's very easy to use go-bindata to compile them into the binary from text files.